src/toast.js

```diff
@@ -10,7 +10,18 @@ const noop = () => false;
  * Merge provided options with the defaults settings and generate the toastId
  */
 function mergeOptions(options, type) {
-  return { ...options, type, toastId: ++toastId };
+  return { ...options, type, toastId: generateToastId(options) };
+}
+
+/**
+ * Generate the toastId either automatically or by provided toastId
+ */
+function generateToastId(options) {
+  if (options === Object(options) && options.toastId === parseInt(options.toastId, 10)) {
```

Comment:
Hey @chuntley,

Could you explain to me what  `options === Object(options) ` does ? Never use that before.

Thanks

Reply:
It's just one of several ways to verify that something is an object. Another option would be: `typeof options === 'object'`.

After looking into it further though, I noticed lodash has a different way of doing it that I would trust more than the others, I'm pushing an update to that check.